.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Test CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
```

Comment:
Please add `- name: Install dependencies` on the line above.
Naming explicitly every step/action ensures that in case of failure we can quickly see which step failed. (for any reason)

Reply:
✅